BlazarService/src/main/java/com/hubspot/blazar/resources/FeedbackResource.java

```diff
@@ -9,35 +11,44 @@
 import com.google.common.base.Optional;
 import com.google.common.collect.ImmutableList;
 import com.google.inject.Inject;
+import com.hubspot.blazar.externalservice.slack.SlackAttachment;
+import com.hubspot.blazar.externalservice.slack.SlackAttachmentField;
+import com.hubspot.blazar.externalservice.slack.SlackMessage;
 import com.hubspot.blazar.base.feedback.Feedback;
-import com.hubspot.blazar.config.BlazarConfiguration;
-import com.hubspot.blazar.config.SlackConfiguration;
-import in.ashwanthkumar.slack.webhook.Slack;
-import in.ashwanthkumar.slack.webhook.SlackAttachment;
+import com.hubspot.blazar.config.BlazarSlackConfiguration;
+import com.hubspot.blazar.integration.slack.SlackClient;
 
 @Path("/feedback")
```

Comment:
I assume this is meant to capture some user feedback. Maybe rename the class to UserFeedback and the end-point /user-feedback